### ISSUE TITLE
No alt text for logo showing in action bar

### ISSUE THREAD

**Issue by poeticabdul** [Automattic/wp-calypso#66091]
### Quick summary

There is no alt text showing for the logo in the action bar even if the image used as the logo has the alt text provided.

![Screenshot at Jul 29 16-53-01](https://user-images.githubusercontent.com/4719037/181807473-20ba1c59-997e-4dda-b55d-350330654d65.png)

### Steps to reproduce

1. Log into a simple site
2. Add an image with alt text as the site logo
3. Visit the site in the same browser and click on the three dots on the action bar 
4. Inspect the element for the logo in the action bar
5. Notice the logo has no alt text

### What you expected to happen

The logo in the action bar should have alt text once the logo has alt text provided.

### What actually happened

The logo in the action bar didn't have an alt text.

### Context

35798797-hc

- [ ] 5434112-zen

### Platform (Simple, Atomic, or both?)

Simple

### Reproducibility

Consistent

### Severity

All

**Comment by jamiepalatnik** [comment]
📌 SCRUBBING
- Tested on Simple ✅ 

📌 FINDINGS/SCREENSHOTS/VIDEO
- I inspected site icon that appears in the actionbar and there is no alt text showing for the site icon in developer tools.

📌 ACTIONS
- Marked as Triaged for Quality Squad review

**Comment by sguzovskaia** [comment]
The customer came back asking about updates on this issue zen-5743494.
